openid-connect-4-credential-issuance-1_0.md

```diff
@@ -191,41 +198,57 @@ Starting point is an interaction of the user with her wallet. The user might, fo
-as part of the issuance flow. 
+(1) (OPTIONAL) obtain credential manifest (as defined in [@DIF.CredentialManifest]) from the issuer with an information of which Verifiable Credentials the Issuer can issue, and optionally what kind of input from the user the Issuer requires to issue that credential.
 
-(1) (OPTIONAL) obtain `credential_manifest` with an information of which VCs the Issuer can issue, and optionally what kind of input from the user the Issuer requires to issue that credential.
+Note: The wallet MAY also obtain the information about the credential issuer's capabilities using other means, which is out of scope of this specification. 
 
-(2) (OPTIONAL) If the issuer expects certain credentials to be presented in the issuance flow (and the wallet contains suitable credentials), the wallet prepares the process by obtaining a nonce from the issuer. This nonce will be used to prevent malicious wallets from being able to replay those presentations. 
+(2) (OPTIONAL) If the issuer expects certain credentials to be presented in the issuance flow, it requests the user to select and confirm presentation of those credentials.
 
-(3.1) In this step, the wallet sends an authorization request to the issuer. This request determines
+(3) If the user has confirmed the presentation of certain credentials with the issuance request, the wallet prepares the process by obtaining a nonce from the issuer. This nonce will be used to prevent malicious wallets from being able to replay those presentations. 
+
+(4) In this step, the wallet sends an authorization request to the issuer. This request determines
 the types of verifiable credentials the wallet (on behalf of the user) wants to obtain. It MAY also
 include verifiable presentations if required by the issuer. 
-Note: the wallet MAY utilize a pushed authorization request to first send the payload of the authorization request to the issuer and subsequently use the `request_uri` returned by the issuer in the authorization
-request. 
-
-(4) The issuer takes over user interface control and interacts with the user. It will authenticate the
-user and ask for her consent to issue the requested credentials. The implementation of this step is at
-the discretion of the issuer and out of scope of this specification. The issuer might, for example, 
-use a local or federated login for that puClientose. It might also utilize verifiable presentations passed 
-in the authorization request or call back to the user's wallet to dynamically obtain verifiable presentations.
-
-(5.1) (OPTIONAL) The issuers calls back to the wallet to fetch verifiable credentials it needs as
-pre-requisite to issuing the requested credentials. From a protocol perspective, the issuers acts
-now as verifier and sends a request as defined in OpenID Connect for Verifiable Presentations 
+
+Note: the wallet MAY utilize a pushed authorization request (see [@!RFC9126]) to first send the payload of 
```

Comment:
encrypting request will blow up the browser..

Reply:
made use of PAR a SHOULD